cmd/krel/cmd/anago/push_git_objects.go

```diff
@@ -104,6 +105,13 @@ func init() {
 		"number of times to retry git operations if a recoverable error occurs",
 	)
 
+	pushGitObjectsCmd.PersistentFlags().BoolVar(
+		&pushGitObjectsOpts.pushMain,
+		"push-main",
+		true,
+		"push changes in the main branch too",
+	)
```

Comment:
I'm wondering why we need this if it defaults to `true` and we do not set it conditionally in `anago`